src/routes/projects.js

```diff
@@ -4,14 +4,20 @@ const router = express.Router();
 const createMemberRequest = require('./controllers/projects/createMemberRequest');
 const followProject = require('./controllers/projects/followProject');
 const getMemberRequests = require('./controllers/projects/getMemberRequests');
+const getSuggestions = require('./controllers/projects/getSuggestions');
 const getProject = require('./controllers/projects/getProject');
+const getAllProjects = require('./controllers/projects/getAllProjects');
```

Comment:
I saw this a while ago, but this could have been part of the GET `/projects?skill[]=value1&tag[]=value2&searchQuery=value3`endpoint. Essentially, no skills, no tags, and empty search queries.